.github/workflows/html-editor-checks.yml

```diff
@@ -0,0 +1,90 @@
+name: Html Editor
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-quill:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Quill repository
+        uses: actions/checkout@v3
+
+      - name: Build package
+        uses: ./.github/actions/quill/steps/build-package
+
+  build:
+    needs: build-quill
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    name: build ${{ matrix.branch }}
+    strategy:
+      fail-fast: false
+      matrix:
+        branch: [22_2, 22_1]
```

Comment:
I didn't find an approach that helps not to duplicate the matrix branch parameter for each job.

@DevExpress/devextreme-devops, maybe you have an idea how to not duplicate this?

I had already tried env variables, but it's not working in this case.
Because I cannot store an array in the env variable :)

Reply:
https://github.com/orgs/community/discussions/26284